templates/about.html

```diff
@@ -0,0 +1,94 @@
+{% extends "_base.html" %}
+{% load compress %} 
+{% load static %}
+
+{% block content %}
+<div class="about-page"> 
+
```

Comment:
Where is the hero section, all pages have a hero section, you need to be consistent